metadata.yml

```
@@ -477,3 +477,13 @@ blogs:
             pull request that we first discussed several weeks ago. Good work
             towards LiveComponents stable!
         category: tech
+
+    "redirect-turbo-fame":
+        title: "Live Components + Turbo Streams: Navigating a Frame"
```

how about:
```suggestion
        title: "Live Components + Turbo Streams: Navigating a Turbo Frame"
```

for clarity.